src/components/navigation/Navigation.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Container, Typography, Link, Divider, Avatar, Popover } from '@mui/material';
+import {
+	NavToolbar,
+	FlexBox,
+	InlineBlock,
+	MBreadcrumbs,
+	PopoverContainer,
+	PopoverButton,
+} from './Navigation.styles';
+
+import CloudIcon from '@mui/icons-material/Cloud';
+import LogoutIcon from '@mui/icons-material/Logout';
+import SettingsIcon from '@mui/icons-material/Settings';
+
+import StorageRoundedIcon from '@mui/icons-material/StorageRounded';
+import { deepPurple } from '@mui/material/colors';
+import { useLogout } from 'hooks/auth/useLogout';
+import { stringAvatar } from 'shared/string-avatar';
+
+const color = deepPurple[700];
+
+export default function Navigation() {
+	const [opened, setOpened] = React.useState<boolean>(false);
+	const avatarContainerRef = React.useRef<HTMLDivElement>(null);
+	const logout = useLogout();
+
+	const handleClick = (): void => setOpened(!opened);
+	const handleClose = (): void => setOpened(false);
+
+	return (
+		<>
+			<NavToolbar>
+				<Container maxWidth={false}>
+					<FlexBox>
+						<FlexBox>
+							<Link
+								underline='none'
```

Comment:
Can we move this styles to *.styles.ts? I think that we should keep only logic and very small quantity of styles in this file.